src/enums.py

```diff
@@ -201,17 +201,21 @@ def as_plant_name(self):
         return self._AS_FTS[self].as_serialised_string().removesuffix("_seed")
 
 
-class Direction(StrEnum):
-    UP = "up"
-    RIGHT = "right"
-    DOWN = "down"
-    LEFT = "left"
+class Direction(IntEnum):
+    UP = auto()
```

Comment:
I suggest you set this value to 0 instead, so the next values can be easily inferred when reading code

Reply:
Done 👍